sdk/keyvault/azure-keyvault-certificates/migration_guide.md

```diff
@@ -0,0 +1,288 @@
+print(certificate.policy.issuer_name)
+```
+
+You can use `get_certificate_version` to retrieve a specific version of a certificate.
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.certificates import CertificateClient
+
+credential = DefaultAzureCredential()
+
+certificate_client = CertificateClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
+
+certificate = certificate_client.get_certificate_version(certificate_name="cert-name", version="cert-version")
+```
+
+### List properties of certificates
+
+In `azure-keyvault` you could list the properties of certificates in a specified vault with the `get_certificates` method. This returned an iterator-like object containing `CertificateItem` instances.
+
+```python
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.keyvault import KeyVaultClient
+
+credentials = ServicePrincipalCredentials(
+    client_id="client id",
+    secret="client secret",
+    tenant="tenant id"
+)
+
+certificate_client = KeyVaultClient(credentials)
+
+certificates = certificate_client.get_certificates(vault_base_url="https://my-key-vault.vault.azure.net/")
+
+for certificate in certificates:
+    print(certificate.attributes.created)
+    print(certificate.x509_thumbprint)
+```
+
+Now in `azure-keyvault-certificates` you can list the properties of certificates in a vault with the `list_properties_of_certificates` method. This returns an iterator-like object containing `CertificateProperties` instances.
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.certificates import CertificateClient
+
+credential = DefaultAzureCredential()
+
+certificate_client = CertificateClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
+
+certificates = certificate_client.list_properties_of_certificates()
+
+for certificate in certificates:
+    print(certificate.name)
+    print(certificate.x509_thumbprint)
+```
+
+### Delete a certificate
+
+In `azure-keyvault` you could delete all versions of a certificate with the `delete_certificate` method. This returned information about the deleted certificate (as a `DeletedCertificateBundle`), but you you could not poll the deletion operation to know when it completed. This would be valuable information if you intended to permanently delete the deleted certificate with `purge_deleted_certificate`.
+
+```python
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.keyvault import KeyVaultClient
+
+credentials = ServicePrincipalCredentials(
+    client_id="client id",
+    secret="client secret",
+    tenant="tenant id"
+)
+
+certificate_client = KeyVaultClient(credentials)
+
+deleted_certificate = certificate_client.delete_certificate(
+    vault_base_url="https://my-key-vault.vault.azure.net/",
+    certificate_name="cert-name"
+)
+
+# wait for the deletion to complete
+
+certificate_client.purge_deleted_certificate(
```

Comment:
```suggestion
# this would fail when deletion hadn't finished
certificate_client.purge_deleted_certificate(
```
i.e. continuing the idea of "the old client had this pitfall we've bridged in the new client"